### ISSUE TITLE
[Windows] Unable to start compiled game due to updated FFmpeg lib

### ISSUE THREAD

**Issue by RattleSN4K3** [UltraStar-Deluxe/USDX#241]
As of merged pull request #235, a compiled version on a Windows system doesn't work due to a required av_frame_free procedure.

Last working commit: a5d290c8e6e8e100f7246c3dd75e3b465738d040
First bugging commit: 10841a83f48602d0009aec0cb7738650907de866

**The entry point for "av_frame_free" cannot be found in "avcodec-57.dll".**

https://github.com/UltraStar-Deluxe/USDX/blob/10841a83f48602d0009aec0cb7738650907de866/src/media/UAudioDecoder_FFmpeg.pas#L285
```
  {$IF LIBAVCODEC_VERSION >= 57000000}
  av_frame_free(@fAudioBufferFrame);
  {$ELSE}
  FreeAlignedMem(fAudioBuffer);
  {$IFEND}
```

**Comment by basisbit** [comment]
indeed, according to https://ffmpeg.org/doxygen/3.1/group__lavu__frame.html#ga979d73f3228814aee56aeca0636e37cc it is part of avutil. Will fix that quickly

**Comment by basisbit** [comment]
fixed by 3168376 .
